src/controller/CHIPDeviceController.cpp

```diff
@@ -947,7 +948,8 @@ DeviceCommissioner::ContinueCommissioningAfterDeviceAttestation(DeviceProxy * de
         return CHIP_ERROR_INCORRECT_STATE;
     }
 
-    if (mCommissioningStage != CommissioningStage::kAttestationVerification)
+    if (mCommissioningStage != CommissioningStage::kAttestationVerification &&
+        mCommissioningStage != CommissioningStage::kAttestationRevocationCheck)
```

Comment:
How would we ever get here in the kAttestationVerification stage?  The whole point of the setup described in https://github.com/project-chip/connectedhomeip/pull/31222#issuecomment-2050392857 is that we do verification, then do the revocation check, then make a _single_ call to the delegate.

Reply:
> How would we ever get here in the kAttestationVerification stage? 

This is part of an existing logic that allows commissioners to bypass device attestation. For conformity, we extended the CommissioningStage check to the new kAttestationRevocationCheck stage.

> The whole point of the setup described in https://github.com/project-chip/connectedhomeip/pull/31222#issuecomment-2050392857 is that we do verification, then do the revocation check, then make a single call to the delegate.

This PR follows the suggestion in the #31222 comment. Firstly by checking the result of VerifyAttestationInformation, then if valid, moves over to the CheckForRevokedDACChain. Please note that the OnDeviceAttestationInformationVerification method is called twice. Second time from the OnDACChainRevocationStatus, which is a callback that is fed to the CheckForRevokedDACChain method.

Reply:
This code seems to be skipping the revocation checks altogether when the DA check fails.  The whole point of the proposal in https://github.com/project-chip/connectedhomeip/pull/31222#issuecomment-2050392857 is that they should _not_ be skipped because:

1. We should have logs showing what happens with revocation even if DA failed.
2. At some later point, not in this PR, we want to change the delegate API to pass in both results so that the delegate has complete information when making its decision.

Reply:
Fixed in the latest commit. Logs shows the revocation state even if DA fails along with the 2 stages of Attestation Verification process.